x-pack/plugins/watcher/server/models/watch/lib/get_interval_type/get_interval_type.test.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getIntervalType } from './get_interval_type';
+
+describe('get interval type', () => {
+  test('should detect fixed intervals', () => {
+    ['1ms', '1s', '1m', '1h', '1d'].forEach((interval) => {
```

Comment:
Nit: how about adding some other units, like '23s' and '5d' here and below to assert that this logic is independent of the quantity value?